app/utils/HsTaskListHelper.scala

```diff
@@ -83,30 +88,47 @@ abstract class HsTaskListHelper(answers: UserAnswers)(implicit messages: Message
 
   private[utils] def addTrusteeHeader(userAnswers: UserAnswers, mode: Mode, srn: Option[String]): Option[SchemeDetailsTaskListSection] = {
     userAnswers.get(HaveAnyTrusteesId) match {
-      case None | Some(true) =>
-        if (userAnswers.allTrusteesAfterDelete.nonEmpty) {
-          Some(
-            SchemeDetailsTaskListSection(
-              Some(isAllTrusteesCompleted(userAnswers)),
-              Link(changeTrusteesLinkText,
-                controllers.register.trustees.routes.AddTrusteeController.onPageLoad(mode, srn).url),
-              None
-            )
-          )
-        } else {
-          Some(
-            SchemeDetailsTaskListSection(None,
-              Link(addTrusteesLinkText,
-                controllers.register.trustees.routes.TrusteeKindController.onPageLoad(mode, userAnswers.allTrustees.size, srn).url),
-              None
-            )
+      case None | Some(true) if userAnswers.allTrusteesAfterDelete.nonEmpty =>
```

Comment:
Might be a bit simpler if you did (userAnswers.get(HaveAnyTrusteesId), userAnswers.allTrusteesAfterDelete.nonEmpty) and then you can lose if checks

Reply:
Thanks, updated